tests/layer_tests/py_frontend_tests/test_torch_decoder.py

```diff
@@ -21,6 +21,7 @@ def get_scripted_model(model):
         model = torch.jit.script(model)
         model.eval()
         model = torch.jit.freeze(model)
+        print(model.inlined_graph)  # will help debugging
```

Comment:
Will we clutter logs this way? May be it is better to comment it out if someone needs